examples/python/gRPC/util/fb_helper.py

```diff
@@ -101,17 +134,47 @@ def getProjectInfo(builder, channel, name):
 
     for i in range(response.ProjectsLength()):
         if response.Projects(i).Name().decode("utf-8") == name:
+            project = response.Projects(i)
             return {
-                "name": response.Projects(i).Name().decode("utf-8"),
-                "uuid": response.Projects(i).Uuid().decode("utf-8"),
-                "frameid": response.Projects(i).Frameid().decode("utf-8"),
-                "geodetic_position": response.Projects(i).GeodeticPosition(),
+                "name": project.Name().decode("utf-8"),
+                "uuid": project.Uuid().decode("utf-8"),
+                "frameid": project.Frameid().decode("utf-8"),
+                "geodetic_position": f"Lat: {project.GeodeticPosition().Latitude()},\
+                    Long: {project.GeodeticPosition().Longitude()},\
+                    Alt: {project.GeodeticPosition().Altitude()}",
```

Comment:
Also use lower case letters for lat, long and alt. 
```suggestion
                "geodetic_position": f"lat: {project.GeodeticPosition().Latitude()},\
                    long: {project.GeodeticPosition().Longitude()},\
                    alt: {project.GeodeticPosition().Altitude()}",
```